pkg/reconciler/ingress/config/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,63 @@
+// +build !ignore_autogenerated
```

Comment:
I'm guessing you copied this.  We should update `./hack/update-codegen.sh`

Reply:
I am not sure what to do here?  Can you elaborate?

Reply:
I don't see a change to `hack/update-codegen` to produce this file, is what I meant.